src/region/RegionAnnotation.tsx

```diff
@@ -4,7 +4,8 @@ import classNames from 'classnames';
 import noop from 'lodash/noop';
 import { getIsCurrentFileVersion } from '../store';
 import { MOUSE_PRIMARY } from '../constants';
-import { Shape, styleShape } from './regionUtil';
+import { Shape } from '../utils/util';
```

Comment:
Time to move Shape to the top-level types? We're importing it in lots of places. Or can we switch to a built-in type, `Required<DomRectInit>`?